content/doc/developer/publishing/requesting-hosting.adoc

```diff
@@ -67,6 +67,18 @@ Follow link:/redirect/continuous-delivery-of-plugins[this guide] for details.
 Jenkins shows plugins in categories in the plugin manager and on the link:https://plugins.jenkins.io/[plugins site].
 The link:../documentation[plugin documentation guide] includes instructions for link:../documentation#documenting-plugins[documenting], categorizing, and link:../documentation#labeling-plugins[labeling] plugins.
 
+== Integrate with Jira
+
+If your chosen issue tracker is Jira, you can setup autolink references, to link from commits directly to the Jenkins Jira issue.
+All you need to do is heading to the `Settings` tab of your GitHub repository and add the following configuration to the `Autolink references` section:
+```txt
+Alphanumeric
+Reference prefix: JENKINS-
+Target URL: https://issues.jenkins.io/browse/JENKINS-<num>
+```
+Commits prefixed with `JENKINS-<num>` will now be linked to the corresponding Jira issue:
+image:../../../content/../images/developer/plugin-development/hosting/jira-auto-reference.png[]
```

Comment:
Can we simplify the image URL?  I built the PR locally with this image URL and it worked as expected.
```suggestion
image:/images/developer/plugin-development/hosting/jira-auto-reference.png[]
```